packages/local-mode-tests/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "local-mode-tests",
+  "version": "1.0.0",
+  "private": true,
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "npx jest --forceExit"
```

Comment:
Is forceExit still needed here?

Reply:
Unfortunately yes. Something about our script that hangs on windows even when we try to kill it. I'll investigate that separately when making a first class api for calling this in tests.